module_evaluations/TCR-46-2024-11-14-mod-linked-data.MD

```diff
@@ -0,0 +1,76 @@
+# Module acceptance criteria template
+
+## Module Name
+mod-linked-data
+
+## How to use this form
+When performing a technical evaluation of a module, create a copy of this document and use the conventions below to indicate the status of each criterion.  The evaluation results should be placed in the [module_evaluations](https://github.com/folio-org/tech-council/tree/master/module_evaluations) directory and should conform to the following naming convention: `{JIRA Key}_YYYY-MM-DD-{module name}.MD`, e.g. `TCR-1_2021-11-17-mod-foo.MD`.  The date here is used to differentiate between initial and potential re-evaluation(s).  It should be the date when the evaluation results file was created.
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/7b10294a5c1c10c7e1a7c5b9f99f04bf07630f06/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+## Administrative
+* [ ] Listed by the Product Council on [Functionality Evaluated by the PC](https://wiki.folio.org/display/PC/Functionality+Evaluated+by+the+PC) with a positive evaluation result.
+  - PC will be reviewing this module on December 12th.
+
+## Shared/Common
+* [X] Uses Apache 2.0 license
+* [X] Module build MUST produce a valid module descriptor
+* [X] Module descriptor MUST include interface requirements for all consumed APIs
+* [X] Third party dependencies use an Apache 2.0 compatible license
+* [X] Installation documentation is included
+  * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
+* [X] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+* [X] Sensitive and environment-specific information is not checked into git repository
+  - Postgres password in docker-compose.yml, but that's standard.
+* [X] Module is written in a language and framework from the [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page[^1]
+* [ ] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+  - Three new library dependencies, these are also in the TCR process currently: 
+    - lib-linked-data-dictionary
+    - lib-linked-data-fingerprint
+    - lib-linked-data-marc4ld
```

Comment:
That's a fair point, you are correct.  But I think the intent of this criterion and the one directly above (about Officially Supported Technologies) is to avoid approving things with new, unapproved dependencies.  Which the dev team [acknowledges in their submission](https://folio-org.atlassian.net/browse/TCR-46) that these libraries are.

> Dependent on: lib-linked-data-marc4ld, lib-linked-data-fingerprint and lib-linked-data-dictionary

Perhaps we need to tweak this criterion (or else to add a new criterion) about dependent libraries also having to be already approved.

In practice, we tend to review backend modules before paired front-end modules to avoid this specific timing situation.   I think we should just review backend libraries before paired backend modules, and then we avoid the same issue.